### ISSUE TITLE
How to scrape multiple videos

### ISSUE THREAD

**Issue by VisateLysang** [stashapp/stash#1353]
#### TL;DR; In short: How to avoid having to paste the URL for each video in order to scrape a gazillion videos?

I want each video in my library to have information pulled from the appropriate site using a community scraper.

As far as I can tell, for each video I have click on the video -> Click edit -> Paste the URL for for the video -> click on the scrape button -> Choose which tags I want to add -> Save.

This would take a tremendous amount of time to do for, say a hundred videos.

Surely there must be a better way? I have read the docs, but I could not find anything that would let me scrape multiple videos automatically.

**Comment by jeremymeyers** [comment]
using the Tagger feature will let you pull from stashbox

**Comment by WithoutPants** [comment]
There is no current functionality for scraping multiple videos. Using the tagger is the best course in the short term. Scraping individually via URL is the next best thing for the moment.
